cmd/sourced/compose/workdir/workdir.go

```diff
@@ -243,3 +272,37 @@ func stripBase(base, target string) (string, error) {
 
 	return filepath.Join("/", p), nil
 }
+
+// Remove removes working directory by removing required files
+// and recursively removes directories up to the workdirs root as long as they are empty
+func Remove(path string) error {
+	workdirsRoot, err := workdirsPath()
+	if err != nil {
+		return err
+	}
+
+	for _, f := range RequiredFiles {
+		if err := os.Remove(filepath.Join(path, f)); err != nil {
+			return errors.Wrap(err, "could not remove from workdir directory")
+		}
+	}
+
+	for {
+		files, err := ioutil.ReadDir(path)
+		if err != nil {
+			return errors.Wrap(err, "could not read workdir directory")
+		}
+		if len(files) > 0 {
```

Comment:
What id there's a `.DS_Store` here?
Why not checking there's no `RequiredFiles`?

Reply:
what if there is `super_import_file_dont_delete_me_or_I_will_loose_1_billion_dollars`?

Reply:
if I understood properly (from prev conversation), that file should not be there, so it would be reasonable to delete it, wouldn't it?

Reply:
same as `.DS_Store` shouldn't be there. But it can be. Better play safe.

Reply:
I understood that `sourced` will ignore user changes on workdirs, not trying to keep them when pruning, but ok :+1: